### PR TITLE
Fix KeyError

### PR DESCRIPTION
**Pull request by inesusvet**
Often row iteration fails with `KeyError` I think this can be prevented.
Not sure if this change will have a negative influence on program's behavior. How should I test this?

### CODE REVIEW

**Comment by inesusvet** [comment]
How to reproduce:
Download a file from ftp://athistory.zerich.com/2018-12-07/AFKS.2018-12-07.Quotes.qsh

Try to pass this file to the example code from readme

Expected behavior:
All rows from the file were iterated

Actual behavior:
`KeyError` is raised

File: qsh/__init__.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def __init__(self, filename=None, mode=None):
         self.fileobj = gzip.open(filename, mode)
 
         signature = self.read(len(self.header.signature.encode("utf8")))
-    
+
```
Comment (inesusvet) [inline qsh/__init__.py:194]: All those changes were made by Atom. I think it's OK

Comment (nethask) [reply qsh/__init__.py:194]: Yeah it's OK!

**Comment by inesusvet** [comment]
Thank you 👍

**Pull request merged**